src/Routes/RosPage/RosPage.js

```diff
@@ -115,6 +116,11 @@ class RosPage extends React.Component {
                 return { label: os, value: os.split(' ')[1] };
             });
 
+            if (osObject.filterValues.items.length === 0) {
+                osObject.filterValues.items = [{ value: '', label: 'No versions available' }];
```

Comment:
@r14chandra just curious to know if we need to pass empty string as `value` for this one?

Reply:
That is not mandatory.

Reply:
okay shall we remove it then @r14chandra ?